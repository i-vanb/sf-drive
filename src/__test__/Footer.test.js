import {shallow} from "enzyme";
import Footer from "../components/Footer";


describe('links', function () {
    it('should be in footer', function () {
        const footer = shallow(<Footer />);
        expect(footer.find('a').length).toBe(3)
    });
    it('snapshot matching', function () {
        const footer = shallow(<Footer />);
        expect(footer).toMatchSnapshot()
    });
});
