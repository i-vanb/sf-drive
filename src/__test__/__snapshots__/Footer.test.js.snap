// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`links snapshot matching 1`] = `
<footer
  className="footer"
>
  <p
    className="footer__rights"
  >
    Â© SkillDrive Inc. 2020
  </p>
  <div
    className="footer__socials"
  >
    <a
      href=""
      rel="nofollow"
      target="_blank"
    >
      <img
        alt="vkontakte link"
        src="1.svg"
      />
    </a>
    <a
      href=""
      rel="nofollow"
      target="_blank"
    >
      <img
        alt="instagram link"
        src="2.svg"
      />
    </a>
    <a
      href=""
      rel="nofollow"
      target="_blank"
    >
      <img
        alt="facebook link"
        src="3.svg"
      />
    </a>
  </div>
</footer>
`;
